shacl12-core/index.html

```diff
@@ -5899,6 +5899,7 @@ <h4>sh:xone</h4>
 								A <a>SHACL list</a> of shapes to validate the value nodes against.
 								<span data-syntax-rule="xone-node">Each value of <code>sh:xone</code> in a shape is a <a>SHACL list</a>.</span>
 								<span data-syntax-rule="xone-members-node">Each <a>member</a> of such list must be a <a>well-formed</a> <a>shape</a>.</span>
+								<span data-syntax-rule="xone-minListLength">There should be at least one <a>member</a> of such list.</span>
```

Comment:
If these are not intended to be [RFC2119 keywords](https://www.w3.org/guide/manual-of-style/#RFC), it's best to not use them, even in lowercase, to avoid common confusion.
```suggestion
								<span data-syntax-rule="xone-members-node">Each <a>member</a> of such list <em title="MUST in RFC 2119 context" class="rfc2119">MUST</em> be a <a>well-formed</a> <a>shape</a>.</span>
								<span data-syntax-rule="xone-minListLength">Each such list <em title="SHOULD in RFC 2119 context" class="rfc2119">SHOULD</em> have at least one <a>member</a>.</span>
```